base/int.jl

```diff
@@ -68,8 +54,14 @@ julia> signed(UInt16)
 Int16
 julia> signed(UInt64)
 Int64
+julia> signed(Int16)
+Int16
+julia> signed(Int64)
+Int64
 ```
 """
+function signed(::Type{<:Integer}) end
```

Comment:
```suggestion
signed(::Type{<:Integer})
```

Reply:
https://docs.julialang.org/en/v1/manual/documentation/index.html#Functions-and-Methods-2
This says first syntax is preferred.